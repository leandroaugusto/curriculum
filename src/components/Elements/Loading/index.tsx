export default function Loading(): React.ReactElement<HTMLParagraphElement> {
  return <p>Loading...</p>;
}
