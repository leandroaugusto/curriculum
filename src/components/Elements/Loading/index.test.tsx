import { render } from "@testing-library/react";
import Loading from ".";

describe("Loading", () => {
  it("renders the Loading element", () => {
    const { container } = render(<Loading />);

    expect(container).toMatchSnapshot();
  });
});
