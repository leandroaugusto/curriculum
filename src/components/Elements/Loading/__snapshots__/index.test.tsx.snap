// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders the Loading element 1`] = `
<div>
  <p>
    Loading...
  </p>
</div>
`;
